packages/templates/cra-template-web-viewer/template/src/App.tsx

```diff
@@ -124,6 +114,7 @@ const App: React.FC = () => {
   );
 
   const onIModelAppInit = useCallback(async () => {
+    // iModel now initialized
```

Comment:
Had to do this to satisfy lint - no empty functions.  Felt this was preferable to an eslint-disable since this is in the template, even though it is a redundant comment :)

Reply:
ah my fault, we will add back the ui providers init before the official release